I'm not able to start a new project with gatsby new command

**> gatsby new my-test-gatsby-site**
```
2016-07-24T12:01:19+0300 <error> new.js:16 () [Error: npm WARN package.json gatsby-starter-default@1.0.0 No repository field.
npm WARN peerDependencies The peer dependency eslint-plugin-jsx-a11y@^1.2.0 included from eslint-config-airbnb will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN deprecated wrench@1.5.8: wrench.js is deprecated! You should check out fs-extra (https://github.com/jprichardson/node-fs-extra) for any operations you were using wrench for. Thanks for all the usage over the years.
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm ERR! Darwin 15.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v4.4.7
npm ERR! npm  v2.15.8
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package eslint-plugin-jsx-a11y@2.0.1 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer eslint-config-airbnb@9.0.1 wants eslint-plugin-jsx-a11y@^1.2.0

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/alexzaporozhets/WebstormProjects/my-test-gatsby-site/npm-debug.log
]
```

Hmmm NPM 2 strikes again. Need to look at our tests again for the new
command.

The simplest solution for now is to uninstall Gatsby, upgrade to npm 3 (npm
install -g npm), and then reinstall Gatsby, and then rerun the "gatsby new"
command.
On Sun, Jul 24, 2016 at 2:12 AM alexzaporozhets <notifications@github.com>
wrote:

> `2016-07-24T12:01:19+0300 new.js:16 () [Error: npm WARN package.json
> gatsby-starter-default@1.0.0 No repository field.
> npm WARN peerDependencies The peer dependency eslint-plugin-jsx-a11y@^1.2.0
> included from eslint-config-airbnb will no
> npm WARN peerDependencies longer be automatically installed to fulfill the
> peerDependency
> npm WARN peerDependencies in npm 3+. Your application will need to depend
> on it explicitly.
> npm WARN deprecated wrench@1.5.8: wrench.js is deprecated! You should
> check out fs-extra (https://github.com/jprichardson/node-fs-extra) for
> any operations you were using wrench for. Thanks for all the usage over the
> years.
> npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or
> higher to avoid a RegExp DoS issue
> npm ERR! Darwin 15.5.0
> npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
> npm ERR! node v4.4.7
> npm ERR! npm v2.15.8
> npm ERR! code EPEERINVALID
>
> npm ERR! peerinvalid The package eslint-plugin-jsx-a11y@2.0.1 does not
> satisfy its siblings' peerDependencies requirements!
> npm ERR! peerinvalid Peer eslint-config-airbnb@9.0.1 wants
> eslint-plugin-jsx-a11y@^1.2.0
>
> npm ERR! Please include the following file with any support request:
> npm ERR!
> /Users/alexzaporozhets/WebstormProjects/my-test-gatsby-site/npm-debug.log
> ]
> `
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gatsbyjs/gatsby/issues/365>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEVh-08xY4KIGxfJKz0iaVIOtLTMSH4ks5qYyx1gaJpZM4JTiXl>
> .
>

--- a/package.json
+++ b/package.json
@@ -97,7 +97,7 @@
     "eslint-plugin-flow-vars": "^0.5.0",
     "eslint-plugin-flowtype": "^2.4.0",
     "eslint-plugin-import": "^1.12.0",
-    "eslint-plugin-jsx-a11y": "^1.5.5",
+    "eslint-plugin-jsx-a11y": "1.5.3",
     "eslint-plugin-react": "^5.2.2",
     "flow-bin": "^0.30.0",
     "iflow-debug": "^1.0.15",

